modulefiles/build.hera.gnu.lua

```diff
@@ -41,7 +41,7 @@ load(pathJoin("w3emc", w3emc_ver))
 sigio_ver=os.getenv("sigio_ver") or "2.3.2"
 load(pathJoin("sigio", sigio_ver))
 
-hdf5_ver=os.getenv("hdf5_ver") or "1.14.0"
+hdf5_ver=os.getenv("hdf5_ver") or "1.14.3_gnu9.2.0"
```

Comment:
Where does this '_gnu9.2.0' come from? Is this one of the system-provided modules?

Reply:
That module load does not appear to do anything. When I remove it (4dca77a), hdf5 v1.14.0 is still loaded.  

I can also remove the netcdf loads from the build module with no effect. Both libraries must be loaded via other modules. 

I will rerun my Gnu tests using 4dca77a.

Reply:
Reran all my Gnu tests. Results did not change: https://github.com/ufs-community/UFS_UTILS/issues/962#issuecomment-2206158835

Reply:
@AlexanderRichert-NOAA - any further comments?